addons/mission/functions/fnc_hunt.sqf

```diff
@@ -26,6 +27,13 @@ params ["_hunters", ["_refresh", 5], ["_hunted", grpNull], ["_searchDistance", 1
 
 if (!isServer) exitWith {};
 
+if (count GVAR(huntGroups) > 6) exitWith {
+    WARNING("The maximum amount of hunt groups has been reached. (6)");
```

Comment:
```suggestion
    WARNING("The maximum amount of hunt groups has been reached (6)!");
```
But this is not actually what it does. This will only exit and throw a warning when you have 7 groups!

Additionally, this should be an error if it's exiting, not a warning.